httpload/cmd/httpload/main_test.go

```diff
@@ -35,3 +36,33 @@ func TestNumberOfRequests(t *testing.T) {
 		t.Fatal("failed to fulfill requests, want:", want, "got:", counter)
 	}
 }
+
+func TestNumberOfRequestsPerSecondPerWorker(t *testing.T) {
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(200)
+		w.Write([]byte("Success!"))
+	}))
+	defer srv.Close()
+
+	flags := flag.NewFlagSet("httpload test", flag.ExitOnError)
+	flags.Usage = func() {
+		progName := os.Args[0]
+		fmt.Fprintf(flags.Output(), "Usage: %s [options...] URL\n", progName)
+		flags.PrintDefaults()
+	}
+
+	args := []string{"-w", "1", "-n", "100", "-q", "25", srv.URL}
+
+	start := time.Now()
+	trueMain(flags, args)
+	elapsed := time.Since(start)
+
+	var maxError time.Duration = 1 * time.Second
+	var wantMin time.Duration = 4 * time.Second
+	var wantMax time.Duration = wantMin + maxError
```

Comment:
Since time.Second is a typed constant (it's `time.Duration`) you don't need to explicitly declare that `maxError` is a `time.Duration`.
Also, by convention, don't use `var` when you declare and assign a value to a variable at the same time.
The semantic of using var is generally when you want to highlight the fact that a variable remains at its default value, for example `var items []int`.
In other words you can simplify these declarations to just:
```suggestion
	maxError := 1 * time.Second
	wantMin := 4 * time.Second
	wantMax := wantMin + maxError
```